python-packages/input4MIPs-CVs/pyproject.toml

```diff
@@ -0,0 +1,24 @@
+[project]
+authors = [
+    { name = "Zebedee Nicholls", email = "zebedee.nicholls@climate-resource.com" },
+    { name = "Paul Durack", email = "durack1@llnl.gov" },
+]
+requires-python = ">=3.9"
+dependencies = [
+    "pandas==2.2.2",
+    "pandas_diff==1.4.7",
+    "typer==0.12.3",
+]
+name = "input4MIPs-CVs"
+version = "0.1.0"  # TODO: make this consistent with everything else
+description = """Python tooling to help with the input4MIPs-CVs repository
```

Comment:
input4MIPs_CVs (underscore not hyphen, but I don't think this is a deal breaker)

Reply:
Either way is fine. The python convention is to use hyphens in the name of the distribution (which is what we have here) and underscores in the python package name (i.e. what you put in `from X import Y`). Apparently this is mostly for legacy reasons, but it is where we are. That's the reason that input4MIPs validation is `input4mips-validation` on PyPI (https://pypi.org/project/input4mips-validation/) but you do `import input4mips_validation` when actually calling it in Python.